### DISCUSSION TITLE
Many Live Photos classified as "Video"

### DISCUSSION THREAD

**Discussion by ethayer** [photoprism/photoprism#3599] [Q&A]
During import, a lot of photos are classified as "videos".  The stack often contains 3-4 files. 
**1** - A video file with the duration of 3-5s (sometimes duration cant be detected by PhotoPrism), 
**2** - a sidecar heic.jpg
**3** - heic file 
**4** (sometimes, but not always) mov.jpg sidecar

It is pretty easy to browse the photos and select Live for the type of photo and this corrects the issue, but is a tedious process.  Is there any way to make this process easier?  PhotoPrism classifies 95% of the live photos as "Live" correctly.

**Comment by lastzero** [comment]
The current time limit for Live Photos is 3 seconds, as specified in the original issue:
- https://github.com/photoprism/photoprism/issues/177#issuecomment-629551485

Has Apple changed the specification recently? If so, we can of course update our code.

**Reply by ethayer** [reply]
I'll do some tests when I get a chance.  I have some suspicions that when you take a photo, then take multiple in a row it does some buffering to the live photo but I'm not certain.  They may not be playing by the rules in their spec.

Scenarios:
Take 1 photo in 3 second time period: Creates an heic and mov with 3 second duration.

Take 2 photos in rapid succession (during 3 second buffer): creates 2 heic images and 2 copies of mov with 3-5 sec duration, book-ended by the video buffer.

Take many photos in rapid (3 second buffer) succession: creates unique (N) heic images per shutter, with N .mov files with entire video buffer duration.

**Reply by lastzero** [reply]
Thank you, much appreciated! Getting file samples for development and testing would be helpful as well:
- https://www.photoprism.app/contact#file-samples
